tests/common/mod.rs

```diff
@@ -121,6 +121,7 @@ pub struct InternalUser<'a> {
     pub age: usize,
     pub address: InternalAddress<'a>,
     pub is_banned: bool,
+    pub α: bool,
```

Comment:
hmm, maybe try β which doesn't seem to be in the list of confusables, to avoid these warnings

Reply:
Good idea.